SpliceAI.pm

```diff
@@ -141,36 +146,56 @@ sub new {
     $output_vcf = 1;
   }
 
+  if(defined($param_hash->{split_output})) {
+    $self->{split_output} = 1;
+  }
+
   return $self;
 }
 
 sub feature_types {
-  return ['Transcript'];
+  return ["Transcript"];
 }
 
 sub get_header_info {
   my $self = shift;
 
   my %header;
 
-  if($output_vcf) {
-    $header{'SpliceAI_pred_SYMBOL'} = 'SpliceAI gene symbol';
-    $header{'SpliceAI_pred_DS_AG'} = 'SpliceAI predicted effect on splicing. Delta score for acceptor gain';
-    $header{'SpliceAI_pred_DS_AL'} = 'SpliceAI predicted effect on splicing. Delta score for acceptor loss';
-    $header{'SpliceAI_pred_DS_DG'} = 'SpliceAI predicted effect on splicing. Delta score for donor gain';
-    $header{'SpliceAI_pred_DS_DL'} = 'SpliceAI predicted effect on splicing. Delta score for donor loss';
-    $header{'SpliceAI_pred_DP_AG'} = 'SpliceAI predicted effect on splicing. Delta position for acceptor gain';
-    $header{'SpliceAI_pred_DP_AL'} = 'SpliceAI predicted effect on splicing. Delta position for acceptor loss';
-    $header{'SpliceAI_pred_DP_DG'} = 'SpliceAI predicted effect on splicing. Delta position for donor gain';
-    $header{'SpliceAI_pred_DP_DL'} = 'SpliceAI predicted effect on splicing. Delta position for donor loss';
+  # Get the SpliceAI tool version from one of the files
+  my $spliceai_version;
+  my $spliceai_file = $self->{_files}[0];
+  open(my $fh, "gzip -dc $spliceai_file |") or die "Could not open '$spliceai_file': $!";
```

Comment:
This line is resulting in `gzip: stdout: Broken pipe` in stdout  although I suspect it is not affecting the output

Reply:
How are you running VEP?